.circleci/config.yml

```diff
@@ -7,17 +7,25 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v5-dependency-cache-{{ checksum "yarn.lock" }}
-            - v5-dependency-cache-
+            - v6-dependency-cache-{{ checksum "yarn.lock" }}
+            - v6-dependency-cache-
       - run: yarn
       - run: echo "Checking if lockfiles changed..." && git diff --exit-code
       - run: npm rebuild node-sass
       - save_cache:
-          key: v5-dependency-cache-{{ checksum "yarn.lock" }}
+          key: v6-dependency-cache-{{ checksum "yarn.lock" }}
           paths:
             - ~/yarn
             - ~/.cache/yarn
             - node_modules
+            - packages/core/node_modules
+            - packages/datetime/node_modules
+            - packages/docs/node_modules
+            - packages/icons/node_modules
+            - packages/labs/node_modules
+            - packages/select/node_modules
+            - packages/table/node_modules
+            - packages/timezone/node_modules
```

Comment:
:( I don't want to list out these package names again like this. let's move back to the old approach of `persist_to_workspace` with `packages/*/node_modules/.bin`

Reply:
If the `singleRun` edit fixes the other branch, I'm happy to just close this whole PR.

Reply:
It worked.